gce/notebook/examples/sea-surface-height.ipynb

```diff
@@ -51,6 +51,19 @@
     "ds"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "### Alternate method to load the data using Intake\n",
+    "### Where the remote YAML spec points to the data on GCS\n",
+    "# import intake\n",
+    "# ds = intake.Catalog('gcs://mdtemp/pangeo.yaml').sea_surface.to_dask()\n",
```

Comment:
Thanks for doing this.

Thoughts on using a file in this github repository instead?  Perhaps in `pangeo/gce/catalog.yaml`?  I suspect that more people within the community are familiar with editing github repositories than modifying files on GCS.

Reply:
That would work fine as well. I don't know where the right place for this would be, so happy to use whatever is convenient.